share/rocm/cmake/ROCMCreatePackage.cmake

```diff
@@ -292,20 +292,31 @@ macro(rocm_create_package)
         set(PYTHON_SITE_PACKAGES
             "/usr/lib/python3/dist-packages;/usr/lib/python2.7/dist-packages"
             CACHE STRING "The site packages used for packaging")
-        foreach(PYTHON_SITE ${PYTHON_SITE_PACKAGES})
-            file(
-                APPEND ${PROJECT_BINARY_DIR}/debian/postinst
-                "
-                mkdir -p ${PYTHON_SITE}
-                echo \"${LIB_DIR}\" > ${PYTHON_SITE}/${PARSE_NAME}.pth
-            ")
-
-            file(
-                APPEND ${PROJECT_BINARY_DIR}/debian/prerm
-                "
-                rm ${PYTHON_SITE}/${PARSE_NAME}.pth
-            ")
-        endforeach()
+        list(GET PYTHON_SITE_PACKAGES 0 PYTHON_SITE_0)
+        list(GET PYTHON_SITE_PACKAGES 1 PYTHON_SITE_1)
```

Comment:
This wont work if `PYTHON_SITE_PACKAGES` has only 1 or 3 paths. This is a cached variable so it can be overwritten by the user to have any number paths. This should still use `foreach` loop to generate the pre/post scripts.

Reply:
Thanks Paul .Will change as per the review comment

Reply:
Changes are made . Pls review

Reply:
Changes are made . Pls review